tasks/schemas.py

```diff
@@ -0,0 +1,50 @@
+from ninja import Schema
+from typing import List, Dict
+from datetime import date, datetime
+from typing import Optional
+
+class TaskSchema(Schema):
+    id: int
+    user_email: str
+    task: str
+    due_by: datetime
+    priority: int
+    is_urgent: Optional[bool] = False
+
+
+class CreateTaskSchema(Schema):
+    user_email: str
+    task: str
+    due_by: datetime
+    priority: int
+    is_urgent: Optional[bool] = False
+
+class UpdateTaskSchema(Schema):
+    user_email: str
+    task: str
+    due_by: datetime
+    priority: int
+    is_urgent: Optional[bool] = False
```

Comment:
These 2 schemas are the same, and only omit the ID of a Task. I think it would be easier to maintain this code if it was something like this:

```suggestion
class TaskIn(ModelSchema):
    class Meta:
        model = Task
        exclude = ["id"]
```

Reply:
will be updated on next update